src/10-static-web-publishing-with-ipfs/index.md

```diff
@@ -0,0 +1,537 @@
+
+The alternative -- self-hosting your content on infrastructure you control -- is not much better. Only the zealous few manage [their own hardware today](http://www.instructables.com/id/Set-up-your-very-own-Web-server/?ALLSTEPS). And those who compromise by hosting their own websites on virtual private servers (VPS) run into a myriad of complex and brittle processes. Even expert web developers can struggle with the complexities of setting up servers, replicating across regions, adding load-balancers, securing machines against hackers, responding to sudden problems, and the constant grind of long-term maintenance. All of this requires large amounts of domain specific knowledge that newcomers find very difficult to use.
+
+> "I just wanted to publish my personal website!"
+
+It is not a mystery why most people use Facebook, Twitter, Medium, and Tumblr to publish and host their public personas, their blogs, and all their thoughts. Web publishing and [digital preservation](https://en.wikipedia.org/wiki/Digital_preservation) are hard.
+
+#### IPFS makes web publishing and hosting easier
+
+This is where IPFS -- the permanent web -- comes in. We've been imagining and implementing vastly easier ways to publish, host, and maintain web content, without sacrificing control or independence.
+
+> - What if it was way easier to publish, host, and maintain web content for the long-term?
+> - What if links didn't break, even when moving hosts?
+> - What if you could count on your links _decades_ from now, with various versions still accessible?
+> - What if "hosting" did not also mean "dependence on disappearing services"?
+> - What if cloud infrastructure worked more like the power grid, where switching providers is transparent?
+> - What if _you_ controlled?
+> - What if names (like DNS domains) mapped straight to content, not servers?
+> - What if we didn't _have to_ rely on centralized naming (DNS)?
+> - What if webapps and websites worked offline, or in local area networks?
+> - What if -- the kicker -- managing all this was as simple as tweeting?
+
+These are all hard problems, and full solutions will take time to build. We don't have everything working yet, IPFS is very young -- we're still implementing fundamental infrastructure and laying the groundwork for user-facing applications. But we have already put dents into many annoying problems, and this post shows off some of the tooling we are building.
```

Comment:
I would remove the `still`, makes it sounds that we are taking more time than promised/expected (which is not actually true)